ShopperInsights/src/main/java/com/braintreepayments/api/ShopperInsightsResult.kt

```diff
@@ -3,6 +3,7 @@ package com.braintreepayments.api
 /**
  * The result object returned when insights about a shopper is requested.
  */
+@ExperimentalBetaApi
```

Comment:
@tdchow I'm inclined to changing this to OptIn instead of the ExperimentalBetaApi annotation.
The result class isn't likely to change. The contents of `ShopperInsightsInfo` are likely to change, but we denote that by annotating that class itself.